packages/server/src/manifest-builder.ts

```diff
@@ -2,6 +2,7 @@ import { bigtestGlobals } from '@bigtest/globals';
 import { Operation } from 'effection';
 import { subscribe } from '@effection/subscription';
 import { once } from '@effection/events';
+import { validateTest } from '@bigtest/suite';
```

Comment:
It's an interesting choice to put the validation logic inside bigtest suite. Is the idea there to make it available to 3rd party tools?

Reply:
I think this could be a nice bonus from this, and I think that'd be pretty nice. The main motivation is so that it can be more easily used once we move this validation to the agent.

Reply:
💯